spring-integration-cassandra/src/main/resources/org/springframework/integration/config/xml/spring-integration-cassandra-1.0.xsd

```diff
@@ -17,5 +17,139 @@
+				Common configuration for cassandra adapters.
+			</xsd:documentation>
+		</xsd:annotation>
+		<xsd:attributeGroup ref="integration:channelAdapterAttributes" />
+		<xsd:attribute name="cassandra-template" type="xsd:string">
+			<xsd:annotation>
+				<xsd:appinfo>
+					<tool:annotation kind="ref">
+						<xsd:documentation>
+						<![CDATA[
+							Reference to an instance of
+							org.springframework.data.cassandra.core.CassandraOperations]]>
+						</xsd:documentation>
+						<tool:expected-type
+							type="org.springframework.data.cassandra.core.CassandraOperations" />
+					</tool:annotation>
+				</xsd:appinfo>
+			</xsd:annotation>
+		</xsd:attribute>
+		<xsd:attribute name="write-options" type="xsd:string">
+			<xsd:annotation>
+				<xsd:appinfo>
+					<tool:annotation kind="ref">
+						<xsd:documentation><![CDATA[
+							Reference to an instance of
+							org.springframework.cassandra.core.WriteOptions]]>
+						</xsd:documentation>
+						<tool:expected-type
+							type="org.springframework.cassandra.core.WriteOptions" />
+					</tool:annotation>
+				</xsd:appinfo>
+			</xsd:annotation>
+		</xsd:attribute>
+		<xsd:attribute name="output-channel" type="xsd:string">
+			<xsd:annotation>
+				<xsd:appinfo>
+					<tool:annotation kind="ref">
+						<xsd:documentation><![CDATA[
+							Reference to an instance of
+							channel where message can be send]]>
+						</xsd:documentation>
+						<tool:expected-type
+							type="org.springframework.messaging.MessageChannel" />
+					</tool:annotation>
+				</xsd:appinfo>
+			</xsd:annotation>
+		</xsd:attribute>
+		<xsd:attribute name="operation-type" use="required">
+			<xsd:annotation>
+				<xsd:documentation><![CDATA[
+							Indicates the operation to do on cassandra db]]></xsd:documentation>
+				<xsd:appinfo>
+					<tool:annotation kind="direct">
+						<tool:expected-type type="java.lang.String" />
+					</tool:annotation>
+				</xsd:appinfo>
+			</xsd:annotation>
+		</xsd:attribute>
+		<xsd:attribute name="cql-ingest" type="xsd:string">
+			<xsd:annotation>
+				<xsd:documentation>
+					<![CDATA[csql query to ingest data]]>
+				</xsd:documentation>
+			</xsd:annotation>
+		</xsd:attribute>
+		<xsd:attribute name="query" type="xsd:string">
+			<xsd:annotation>
+				<xsd:documentation>
+					<![CDATA[statement to use in prepared statement]]>
+				</xsd:documentation>
+			</xsd:annotation>
+		</xsd:attribute>
+        <xsd:attribute name="produces-reply" default="false" type="xsd:boolean">
+                <xsd:annotation>
+              <xsd:documentation>
+                <![CDATA[statement to use in prepared statement]]>
+              </xsd:documentation>
+            </xsd:annotation>
+       </xsd:attribute>
+    
+	</xsd:complexType>
+	
+	<xsd:complexType name="queryParameterType">
+		<xsd:annotation>
+			<xsd:documentation><![CDATA[
+				Expression to be evaluated against the Message to replace a query Parameter {placeholder} with the evaluation result.
```

Comment:
We don't replace `{placeholder}`, we use there standard CQL named parameters discovery.
See the sample:
```
SELECT * FROM book WHERE author = :author limit :size
```